src/Sarif/Core/Stack.cs

```diff
@@ -102,7 +102,7 @@ public static Stack Create(string stackTrace)
 
             stack.Frames = new List<StackFrame>();
 
-            var regex = new Regex(StackFrame.AT + @"([^)]+\))(" + StackFrame.IN + "([^:]+:[^:]+)" + StackFrame.LINE + " (.*))?", RegexOptions.Compiled);
+            var regex = new Regex(StackFrame.AT + @"([^)]+\))(" + StackFrame.IN + @"([^:]+:[^:]+|\/.+)" + StackFrame.LINE + " (.*))?", RegexOptions.Compiled);
```

Comment:
>+|\/.+ [](http://example.com/codeflow?start=95&length=6)

I am confused by this regex.

Reply:
the issue was, Regex checking path with ":", expecting like `c:\path`, it will only work in Windows. in Linux I run and see it starts with / like `/aaa.cs`

according to your  other comment I have reverted.

`Perhaps this specific issue warrants its own PR?
Agree, created one and reverted this change.
[https://github.com/microsoft/sarif-sdk/issues/2451`](https://github.com/microsoft/sarif-sdk/issues/2451`)